lib/engine/game/g_2038/map.rb

```diff
@@ -0,0 +1,205 @@
+            'code' => 'city=revenue:green_30|brown_40;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=I',
+          },
+          '2004' =>
+          {
+            'count' => 4,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_40|brown_50;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=I',
+          },
+          '2005' =>
+          {
+            'count' => 8,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_50|brown_60;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=I',
+          },
+          '2006' =>
+          {
+            'count' => 2,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_20|brown_50;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=R',
+          },
+          '2007' =>
+          {
+            'count' => 4,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_30|brown_60;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=R',
+          },
+          '2008' =>
+          {
+            'count' => 7,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_40|brown_70;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;'\
+                      'path=a:4,b:_0;path=a:5,b:_0;label=R',
+          },
+          '2009' =>
+          {
+            'count' => 7,
+            'color' => 'yellow',
+            'code' => 'city=revenue:green_20|brown_60;city=revenue:green_20|brown_60;path=a:0,b:_0;path=a:0,b:_1;'\
+                      'path=a:1,b:_0;path=a:1,b:_1;path=a:2,b:_0;path=a:2,b:_1;path=a:3,b:_0;path=a:3,b:_1;'\
+                      'path=a:4,b:_0;path=a:4,b:_1;path=a:5,b:_0;path=a:5,b:_1;label=N/N',
+          },
+          # TODO: Fill in the rest of these once confirmed (also note that all these lanes should probably be double)
+          '2010' =>
+          {
+            'count' => 2,
+            'color' => 'yellow',
+            'code' => 'town=revenue:yellow_30|gray_60;town=revenue:20;town=revenue:20;path=a:1,b:_0;path=a:2,b:_0;'\
+                      'path=a:4,b:_1;path=a:5,b:_1;path=a:0,b:_2;path=a:3,b:_2',
+          },
+          '2011' =>
+          {
+            'count' => 1,
+            'color' => 'yellow',
+            'code' => 'city=revenue:yellow_30|gray_60,slots:2;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:4,b:_0;'\
+                      'path=a:5,b:_0;label=I',
+          },
+          '2012' =>
+          {
+            'count' => 1,
+            'color' => 'yellow',
+            'code' => 'city=revenue:80,slots:4;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:4,b:_0;'\
+                      'path=a:5,b:_0;label=I',
```

Comment:
Is there any difference between tiles 2012 to 2022?

Reply:
Addressed in comment above.